### ISSUE TITLE
Crash with Lootbags 

### ISSUE THREAD

**Issue by Rubadoo** [Malorolam/LootBags#186]
[crash-2019-05-29_19.06.29-server.txt](https://github.com/Malorolam/LootBags/files/3234911/crash-2019-05-29_19.06.29-server.txt)

**Comment by Malorolam** [comment]
Did you remove all bags from being able to be stored?

**Comment by Rubadoo** [comment]
nope, do you mean in the configs?

**Comment by Malorolam** [comment]
Yes, that crash would only happen if the list of bags that could be removed from the storage is empty. The most likely way that would happen is if the bag config was set up so that no bags could be removed from storage.

**Comment by Rubadoo** [comment]
I havent changed the config at all which is weird. Should I delete the config or something to fix it?

**Comment by Malorolam** [comment]
Post the Lootbags_BagConfig so I can see it.
